data/core.yaml

```diff
@@ -2066,8 +2066,8 @@ en:
       tag_this_as_lower:
         title: Tag this as lower
       upgrade_tags:
-        title: Upgrade the tags
-        annotation: Upgraded old tags.
+        title: Change the tags
+        annotation: Changed old tags.
```

Comment:
There’s already another action with a similar title:

https://github.com/openstreetmap/iD/blob/7d6dda002b53efcf3574f49ea9a37eff7800c6d1/data/core.yaml#L86-L87

Instead of “upgrade the tags”, how about “accept the suggested tags”, since that’s what the diff below calls them anyways?

Reply:
> There’s already another action with a similar title:

do you know here it is used?

> accept the suggested tags

I would refer to go further away from implying it is a good change, but this is also improvement compared to the current state

Reply:
`change_tags` is when you manually change a tag. So if you twiddle one of the fields, the Undo button’s tooltip will say “Undo: Changed tags.”

The other potential source of confusion is that we’d wind up with two options for changing the tags. The second option indicates the effect of the change, but the first does not:

<img src="https://github.com/user-attachments/assets/1e7eed7d-d374-48a8-9184-6528482f840d" width="359" alt="Change the tags. Tag as not the same “Volvo”.">

“Accept the suggested updates” would match the line “Suggested updates” below. But if you find that too optimistic, then we’d need to come up with a replacement for that string too. Personally, I don’t find it too leading, as long as the problem statement in #11029 maintains transparency with the user.

Reply:
changed to this one then

note: I think it is still not making clear that this automated changes may be damaging, but better than what we have now

note: I am not native speaker, I cannot judge is it Engrish or English